sdcm/k8s_configs/eks/scylla-node-prepare.yaml

```diff
@@ -55,6 +55,105 @@ subjects:
   name: node-setup-daemonset
   namespace: default
 ---
+# Daemonset that will change cpuManagerPolicy to static.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cpu-policy
+spec:
+  selector:
+    matchLabels:
+      name: cpu-policy
+  template:
+    metadata:
+      labels:
+        name: cpu-policy
+    spec:
+      hostPID: true
+      hostIPC: true
+      serviceAccountName: node-setup-daemonset
+      containers:
+      - name: cpu-policy
+        image: bitnami/kubectl:1.21.4
+        imagePullPolicy: Always
+        env:
+          - name: NODE
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: HOSTFS
+            value: /mnt/hostfs
+          - name: KUBELET_CONFIG_PATH
+            value: /etc/kubernetes/kubelet/kubelet-config.json
+        securityContext:
+          privileged: true
+          runAsUser: 0
+        volumeMounts:
+          - name: hostfs
+            mountPath: /mnt/hostfs
+            mountPropagation: Bidirectional
+        command:
+          - "/bin/bash"
+          - "-c"
+          - "--"
+        args:
+          - |
+            set -ex
+
+            TOKEN_PATH=$(find / -name token | grep node-setup-daemonset -m1)
+            TOKEN=$(cat $TOKEN_PATH)
+            CA_CRT=$TOKEN_PATH/../ca.crt
+            kubectl config set-cluster scylla --server=https://kubernetes.default --certificate-authority=$CA_CRT
+            kubectl config set-credentials qa@scylladb.com --token=$TOKEN
+            kubectl config set-context scylla --cluster=scylla --user=qa@scylladb.com
+            kubectl config use-context scylla
+
+            echo "Change kubelet config and restart it's service"
+            if [[ `grep "cpuManagerPolicy" $HOSTFS$KUBELET_CONFIG_PATH | wc -l` -eq 0 ]]
+            then
+              kubectl drain $NODE --force --ignore-daemonsets --delete-local-data --grace-period=60
+              awk 'NR==2{print "  \"cpuManagerPolicy\": \"static\","}1' $HOSTFS$KUBELET_CONFIG_PATH > tmp-kubelet-config.json \
+                && cat tmp-kubelet-config.json > $HOSTFS$KUBELET_CONFIG_PATH
+            elif [[ `grep "cpuManagerPolicy" $HOSTFS$KUBELET_CONFIG_PATH | wc -l` -gt 1 ]]
+            then
+              kubectl drain $NODE --force --ignore-daemonsets --delete-local-data --grace-period=60
```

Comment:
This one will hit only when there are two or more lines with 'cpuManagerPolicy', which is never.

Reply:
Done